web/src/classic/components/molecules/Dashboard/Project.tsx

```diff
@@ -47,7 +49,8 @@ const Project: React.FC<Props> = ({ className, project }) => {
           </Text>
           <Actions isHovered={isHovered}>
             <ButtonWrapper>
-              <StyledLink to={`/edit/${sceneId}`}>
+              <StyledLink
+                to={project.projectType === "beta" ? `/scene/${sceneId}` : `/edit/${sceneId}`}>
```

Comment:
@nina992 
```suggestion
                to={project.projectType === "beta" ? `/scene/${sceneId}/scene` : `/edit/${sceneId}`}>
```

I changed the beta path by [another PR](https://github.com/reearth/reearth/pull/448).
Please make sure the link is working correctly after this branch is updated.